app/data/helpers/generators.js

```diff
@@ -0,0 +1,72 @@
+
+
+const faker = require('faker')
+faker.locale = 'en_GB'
+
+exports.firstName = (sex) => {
+  if(sex === 'male') {
+    return faker.helpers.randomize([
+      'Tony',
+      'Thor',
+      'Henry',
+      'Robert',
+      'Steve',
+      'Clinton',
+      'Pietro',
+      'Jacques',
+      'Harry',
+      'Luke',
+      'Victor',
+      'Samuel'
+    ])
+  } else {
+    return faker.helpers.randomize([
+      'Janet',
+      'Wanda',
+      'Natasha',
+      'Heather',
+      'Patricia',
+      'Carol',
+      'Jennifer',
+      'Monica',
+      'Barbara',
+      'Bonita',
+      'Susan',
+      'Julia'
+    ])
+  }
+}
+
+exports.lastName = (sex) => {
```

Comment:
Does the last name need to be gender-specific?

Reply:
And can you sort the list for ease on the eyes?

Reply:
okay